app/scripts/components/common/mapbox/layers/raster-paint-layer.tsx

```diff
@@ -0,0 +1,114 @@
+import { useEffect, useMemo } from 'react';
```

Comment:
🔧 : We are working to deprecate this path. The new path for this logic should go under `/common/map/`. I believe you would want to migrate this over to `/common/map/style-generators/`.